packages/core-api/src/app/types.ts

```diff
@@ -59,6 +60,11 @@ export type AppComponents = {
   BootErrorPage: ComponentType<BootErrorPageProps>;
   Progress: ComponentType<{}>;
   Router: ComponentType<{}>;
+  ErrorBoundaryFallback: ComponentType<
+    ErrorBoundaryFallbackProps & {
+      plugin: BackstagePlugin;
```

Comment:
make this optional - we want to be able to place one at the app root as well probably